template/razor_repl_expr.go

```diff
@@ -14,6 +14,7 @@ import (
 const (
 	protectString      = "_=LONG_STRING="
 	literalAt          = "_=!AT!=_"
+	literalTrippleBTs  = "_=!TRIPPLE_BT!=_"
```

Comment:
typo. `triple` Also, `backticks` in longform would be clearer, all I could think of was Bluetooth :smile: 
`_=!TRIPLE_BACKTICKS!=_`